common/beerocks/bcl/source/beerocks_ucc_listener.cpp

```diff
@@ -350,6 +350,23 @@ bool beerocks_ucc_listener::get_send_1905_1_tlv_hex_list(
     return false;
 }
 
+static void set_relay_indicator(std::shared_ptr<ieee1905_1::cCmduHeader> &cmdu_header)
```

Comment:
This should be done directly in `CmduMessageTx::create`, not here in the listener, so we are sure we don't ever forget to set the relay indicator correctly, not only in the UCC listener. This way you will also take care of the destination (multicast) mac address transparently instead of having to manually set it whenever you create such a CMDU.

Reply:
Good idea.